docs/documentation/using-notify.md

```diff
@@ -84,10 +84,10 @@ example:
       <form class="form" method="post">
 
         <div class="govuk-form-group">
-          <label class="govuk-label" for="email-address">
+          <label class="govuk-label" for="email">
             Email address
           </label>
-          <input class="govuk-input" id="email-address" name="emailAddress" type="text">
```

Comment:
If we're changing `name` I think we'll have to update the rest of the doc to use the new name.

Reply:
I think it probably makes more sense to keep the name - fewer "email" strings for a user to parse.